api.go

```diff
@@ -32,11 +33,86 @@ func (b *Bot) Raw(method string, payload interface{}) ([]byte, error) {
 	defer resp.Body.Close()
 
 	json, err := ioutil.ReadAll(resp.Body)
+	data := apiErrorRx.FindStringSubmatch(string(json))
 	if err != nil {
 		return []byte{}, wrapSystem(err)
 	}
 
+	if data == nil {
+		return json, nil
+	}
+
+	description := data[2]
+	code, _ := strconv.Atoi(data[0])
+	switch description {
+	case ErrUnauthorized.ʔ():
```

Comment:
The impeccable use of `ʔ` sign is a in itself showing you are a man of high creativity and high potential. Very good choice style-wise, my friend!